psalm.xml

```diff
@@ -15,13 +15,4 @@
             <directory name="vendor" />
         </ignoreFiles>
     </projectFiles>
-
-    <issueHandlers>
-        <!-- This check is confused by Symfony's "mixed" property types. -->
-        <ReservedWord>
-            <errorLevel type="suppress">
-                <directory name="." />
-            </errorLevel>
-        </ReservedWord>
-    </issueHandlers>
```

Comment:
As soon as Symfony 6 is stable, Psalm will fail again. Let's keep the issue handler in here until we have a better solution.

Reply:
I thought this was because we were using bleeding edge Psalm?